packages/web/public/static/locales/zh_CN/common.json

```diff
@@ -46,6 +46,8 @@
     "bodyOne": "需要帮助？我们在这里！",
     "bodyTwo": "请发送邮件至 ",
     "bodyThree": " 并详细描述您的问题，我们将尽快回复！",
+    "bodyFour": "Have you seen concerning behavior on the platform? Please refer to our Terms of Service and always feel free to report anything you feel may be misaligned with our guidelines:",
+    "termsOfService": "Terms of Service",
```

Comment:
Hi @Harper-Tan ! Would you be able to translate these whenever you have the time? I added them to `zh_TW` as well which you can see below.

I think for now we don't need to worry about translating the actual Terms of Service document because it could be a lot of work and will likely evolve quite a lot as we learn more. I'm thinking perhaps once we have a very stable version of the document that has been checked by professionals, then we could go about translating it into our UI languages.